package.json

```diff
@@ -44,9 +44,11 @@
   },
   "homepage": "https://github.com/multiformats/js-multibase#readme",
   "dependencies": {
-    "bs58": "^3.0.0"
+    "base-x": "1.0.4",
+    "bs58": "^3.0.0",
```

Comment:
No, indeed we don't need it, but several test cases still rely on that package (and I didn't want to change the test cases before a first code review) I'll remove it while I refactor the existing bs58 dependent tests (and I'll add new test cases accordingly)